ui/src/components/WMWorkerPoolEditor/index.jsx

```diff
@@ -315,10 +332,57 @@ export default class WMWorkerPoolEditor extends Component {
       workerPool.config !== JSON.stringify(config || {}, null, 2) ||
       joinWorkerPoolId(workerPool.workerPoolId1, workerPool.workerPoolId2) !==
         workerPoolId;
+    const { provisionerId, workerType } = splitWorkerPoolId(workerPoolId);
 
     return (
       <Fragment>
         <ErrorPanel fixed error={error} />
+        {!isNewWorkerPool && (
+          <Paper component="ul" className={classes.overviewList}>
+            <li>
+              <Chip
+                size="small"
+                className={classes.metadata}
+                avatar={<Avatar>{currentCapacity || 0}</Avatar>}
+                label="Current Capacity"
+              />
+            </li>
+            <li>
+              <Chip
+                size="small"
+                className={classes.metadata}
+                avatar={<Avatar>{pendingTasks || 0}</Avatar>}
+                label="Pending Tasks"
+              />
+            </li>
+            <li>
+              <Chip
+                size="small"
+                className={classes.metadata}
+                icon={<WorkerIcon />}
+                label="Workers (Queue View)"
+                component="a"
```

Comment:
We have a custom component "Link" that allows prefetching the view before the user clicks on it. It will help make the transition much faster. We would need to also change the `href` to `to`.

Reply:
See the diff :)